libexec/bootstrap-scripts/functions

```diff
@@ -101,7 +101,10 @@ get_section() {
             else
                 # We already found it, still in section
                 if [[ "$FOUND" == "yes" ]]; then
-                    echo $line
+                    if [ ! -z "${line}" ]; then
```

Comment:
I have some questions about this line.  

First, isn't a `! -z` just a `-n`?  Should probably use that instead.

But second, I question why this line is being added at all.  This is going to strip empty lines right?  Don't we want to keep them.  In fact there was a recent PR to ensure that we keep them.  https://github.com/bleuchien/singularity/blame/df5e815f812c1260f59b694c1771c1efbc261d98/libexec/bootstrap-scripts/functions#L78

Reply:
Sorry for the slow response:
- I removed the additional line from the changelog
- rearranged the contributors file entry
- and removed the empty line filter as requested